src/backend/migrations/20220726000000_users.js

```diff
@@ -0,0 +1,13 @@
+
+exports.up = knex => knex.schema.createTable('users', table => {
+    table.string('id'); // P-key
+    table.string('given_name');
+    table.string('family_name');
+    table.string('email');
```

Comment:
Is it possible to chain this together with `unique(email)` ?

Reply:
Can just do `table.string('email').unique()`